README.rst

```diff
@@ -221,6 +221,10 @@ German (BETA, decimals are not supported yet):
     >>> alpha2digit(text, "de", ordinal_threshold=0)
     'Es ist ein Buch mit 3000 Seiten aber nicht das 1..'
 
+    >>> text = "Pi ist drei Komma eins vier und so weiter, aber nicht drei Komma vierzehn :-p"
+    >>> alpha2digit(text, "de", ordinal_threshold=0)
+    'Pi ist 3,14 und so weiter, aber nicht 3 Komma 14 :-p'
```

Comment:
(simple curiosity) Really? can't you say "*drei Komma vierzehn*" in German? You must spell the decimals unit by unit?

Reply:
Well, you can do it and I think 50 years ago it wasn't uncommon. At least my father still uses it from time to time for 2 digits (not my mother though, she was a mathematician ^^), but in school we learned to avoid it and in science it always feels like a crime to me :-p. I'm not a fan and it was (a bit) easier to ignore it so I decided not to support it ;-).
If someone wants to implement it they should keep in mind that certain combinations don't work like "3 komma 14 17" or "3 komma 1 14" and so on ... and probably no one would say "3 komma 14156" ^^.

Reply:
Thanks for this explanation! Looks like a "recent" cultural shift, fascinating!

Reply:
Indeed :slightly_smiling_face: